src/modals/storage-connections/StorageConnectionsDataProvider.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import getPluginApi from '_/plugin-api'
+import { engineGet, enginePost, enginePut, engineDelete } from '_/utils/fetch'
+import DataProvider from '_/components/helper/DataProvider'
+import { webadminToastTypes } from '_/constants'
+import { msg } from '_/intl-messages'
+
+const createConnection = async (connection) => {
+  const response = await enginePost(
```

Comment:
Why not just `return await enginePost(...)`? The same for other post/delete api calls.

Reply:
Good point